File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnStudioPickCompId.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnCompType;
import com.neome.java.api.meta.base.Types.MetaIdComposite;
import com.neome.java.api.meta.base.Types.MetaIdForm;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnStudioPickCompId extends DefnFieldEditable
{
  @Nullable
  public MetaIdComposite[] excludeCompositeIdSet;

  @Nullable
  public EnumDefnCompType[] filterCompTypeSet;

  public MetaIdForm formId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnStudioPickCompId.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.EnumDefnCompType
import com.neome.api.meta.base.Types.MetaIdComposite
import com.neome.api.meta.base.Types.MetaIdForm

interface DefnStudioPickCompId : DefnFieldEditable
{
  val excludeCompositeIdSet: Array<MetaIdComposite>?
  val filterCompTypeSet: Array<EnumDefnCompType>?
  val formId: MetaIdForm
}